README.md

```diff
@@ -981,7 +981,7 @@ Run `sls deploy`, the defined Stepfunctions are deployed.
 
 ## IAM Role
 
-The IAM roles required to run Statemachine are automatically generated. It is also possible to specify ARN directly.
+The IAM roles required to run Statemachine are automatically generated for the state machines lambda, with the policy name of `StatesExecutionPolicy-<environment>`. This is given the default permissions of allowing the lambda InvokeFunction. You can also specify a custom ARN directly to the step functions lambda.
```

Comment:
Please rephrase "state machines lambda" to just "state machine" and add a mention that an IAM role is generated for each state machine in the `serverless.yml` and the permissions are tailored to the services that state machine integrates with - Lambda, SNS, SQS, etc (it's not just Lambda `InvokeFunction`).

Also, when you said `policy name` did you mean `IAM role name`? That format has changed since 2.14.0 when we moved to 1 IAM per state machine.

Reply:
Yes should have been IAM role name, sorry for the confusion.